proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -222,7 +222,7 @@ func (e *ExecutionPayload) UnmarshalJSON(enc []byte) error {
 	if err != nil {
 		return err
 	}
-	e.BaseFeePerGas = bytesutil.PadTo(baseFee.Bytes(), fieldparams.RootLength)
```

Comment:
`ExecutionPayload.BaseFeePerGas` (L183 as well) should always be big-endian  to match `ExecutionBlock.BaseFeePerGas`, and spec. It seems like this is reverting 5566b2bb1570f4d3bc2515200eba8e4a40d19bce?.